File: src/main/java/ar/edu/utn/frba/dds/domain/lugares/Coordenada.java
package ar.edu.utn.frba.dds.domain.lugares;

import lombok.*;

import javax.persistence.Column;
import javax.persistence.Embeddable;

@Getter
@Setter
@Builder
@Embeddable
@NoArgsConstructor
@AllArgsConstructor
public class Coordenada {

    @Column(name = "latitud")
    private Double latitud;

    @Column(name = "longitud")
    private Double longitud;

}

File: RecomendadorDeDonacion - Servicio TP anual/RecomendadorDeDonacion/RecomendadorDeDonacion/Domain/Coordenada.cs
namespace RecomendadorDeDonacion.Domain
{
    public class Coordenada
    {
        public double latitud { get; set; }
        public double longitud { get; set; }

        public Coordenada(double latitud, double longitud) { 
            this.latitud = latitud;
            this.longitud = longitud; 
        
        }
    }
}
